living_documentation_generator/generator.py

```diff
@@ -10,6 +10,7 @@
 from living_documentation_generator.github_projects import GithubProjects
 from living_documentation_generator.model.config_repository import ConfigRepository
 from living_documentation_generator.model.consolidated_issue import ConsolidatedIssue
+from living_documentation_generator.model.github_project import GithubProject
```

Comment:
You are not using this via IDEA.

Reply:
Fixed in commit - cd38013904714dffc28f5b1d1b5c04546fd4bfae.